src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -231,8 +232,27 @@ def __discover_nodes(self):
                 exc_info=True,
             )
 
+    def __update_cluster_status(self):
+        if self.telemetry is None:
+            return
+        try:
+            cluster_status = ClusterStatus(
+                cluster_id= self.cluster_name,
+                version= self.telemetry.get("runner_version"),
+                last_alert_at= self.telemetry.get("last_alert_at"),
+                account_id= self.account_id
+            )
+
+            self.dal.publish_cluster_status(cluster_status)
```

Comment:
Regarding this issue. I think setting for a larger interval is good. 
Problem is I'm afraid we might miss the first prometheus alert and the UI will show error for a while.
I want to find an elegant general solution to the detect some change in the telemetry.
can we postpone that fix for the next telemetry issue?

Reply:
hmm
What if we saved first_alert_time as well
And send every short period, only if it's not empty, and only up to 10 min passed since first_alert_time.
Will that work?

Reply:
ok did something in that flavor